common/djangoapps/student/models.py

```diff
@@ -1966,7 +1978,7 @@ def dynamic_upgrade_deadline(self):
             return None
 
         try:
-            if not self.schedule or not self.schedule.enrollment.is_active:  # pylint: disable=no-member
+            if not self.schedule or not self.schedule.active:  # pylint: disable=no-member
```

Comment:
What's the reason for this change?

Reply:
this is a typo. Still getting used to my editor, so might have accidentally done this